doc/api/stream.markdown

```diff
@@ -413,8 +413,9 @@ then the data will be encoded in hexadecimal string format.
 
 This properly handles multi-byte characters that would otherwise be
 potentially mangled if you simply pulled the Buffers directly and
-called [`buf.toString(encoding)`][] on them. If you want to read the data
-as strings, always use this method.
+called [`buf.toString(encoding)`][] on them. Also you can disable 
+any encoding at all with `readable.setEncoding(null)`.
+If you want to read the data as strings, always use this method.
```

Comment:
I would flip these last two sentences around, or move the new sentence to a separate paragraph.